cylc/flow/cfgspec/workflow.py

```diff
@@ -1334,6 +1336,27 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     "[platforms][<platform name>]submission retry delays"
                 )
             )
+            Conf(
+                'run mode', VDR.V_STRING,
+                options=list(RunMode.OVERRIDING_MODES.value),
+                default=RunMode.LIVE.value,
+                desc=f'''
+                    For a workflow run in live mode run this task in skip
+                    mode.
+
+                    {RunMode.LIVE.value}:
+                        {RunMode.LIVE.__doc__}
+                    {RunMode.SKIP.value}:
+                        {RunMode.SKIP.__doc__}
```

Comment:
Almost:

![Screenshot from 2024-08-30 13-13-48](https://github.com/user-attachments/assets/dcb4b2c0-f664-4fd2-8897-1f19838bbcc0)

Attributes can't have docstrings in Python, you'll need to use a method like we do with `StopMode` e.g. `RunMode.LIVE.describe()`

Reply:
Pity - I misssed the `StopMode.describe` method and thought that this was some nice magic. 😢